packages/hathor-rpc-handler/eslint.config.js

```diff
@@ -7,9 +7,6 @@ export default tseslint.config(
   eslint.configs.recommended,
   ...tseslint.configs.recommended,
   {
-    ignores: [
-      'dist/**/*.js',
-    ],
```

Comment:
Added the ignore on `package.json`

Reply:
For some reason eslint was ignoring this, I didn't dive much deeper into figuring out why, so I just moved it to package.json